### ISSUE TITLE
[Feature request]better armor-value lock.

### ISSUE THREAD

**Issue by wootaboot** [Coders-After-Dark/CompatSkills#103]
the armor-value lock feature does not take into account that different pieces of the same set have different values. i.e diamond armor boots and diamond armor chest would have very different requirements because chest gives more armor. just want to have a feature that takes that into consideration due to the randomness of armor making with TiC armor.

**Comment by pupnewfster** [comment]
(Transfered this to CompatSkills issue tracker)

I will leave this open as a feature request for a way to lock by defense level. But the armor value locking works as intended, in fact I made the armor lock value with the exact purpose of using it with constructs armory, so as to dynamically lock things based on their combined armor values.

**Comment by pupnewfster** [comment]
I don't know if it is stored in NBT as back when I used constructs armory I don't believe it had a "defense" level, but if it is stored somewhere in the NBT of the item, you can use a partial NBT lock to match specific "defense" levels.

**Comment by wootaboot** [comment]
defense is stored in NBT so ill try to get that to work, thanks.

**Comment by wootaboot** [comment]
seems it works but it has to be very specific. Any way to make it so that its greater then or equal to number for NBT tags?

**Comment by pupnewfster** [comment]
Nope, best you could do is if they are all whole numbers just loop over a bunch of values adding them for each.

**Comment by wootaboot** [comment]
well dont know if they are always integers but i think that there is a finite number of possibilities so ill probably just make a function to hit them all. Thanks for all the help.
